### ISSUE TITLE
Issue with timestamp_create method. 

### ISSUE THREAD

**Issue by tameem92** [sporkmonger/uuidtools#41]
Hey, 

I am receiving the following error on Jessie os using the timestamp_create method. 

```
[1] pry(main)> UUIDTools::UUID.timestamp_create
Usage: ip [ OPTIONS ] OBJECT { COMMAND | help }
       ip [ -force ] -batch filename
where  OBJECT := { link | addr | addrlabel | route | rule | neigh | ntable |
                   tunnel | tuntap | maddr | mroute | mrule | monitor | xfrm |
                   netns | l2tp | tcp_metrics | token | netconf }
       OPTIONS := { -V[ersion] | -s[tatistics] | -d[etails] | -r[esolve] |
                    -f[amily] { inet | inet6 | ipx | dnet | bridge | link } |
                    -4 | -6 | -I | -D | -B | -0 |
                    -l[oops] { maximum-addr-flush-attempts } |
                    -o[neline] | -t[imestamp] | -b[atch] [filename] |
                    -rc[vbuf] [size]}
NoMethodError: undefined method `split' for nil:NilClass
from /usr/local/bundle/gems/uuidtools-2.1.4/lib/uuidtools.rb:611:in `first_mac'
```

I have looked into it in a bit of detail, Jessie uses ip instead of ifconfig and the code for the gem does the following: 

`all_switch = all == nil ? "" : "-a"  `

The "-a" options seem to be incorrect options for ip service. Should be 'device a'

```
root@651fc07a3a3a:r# ip -a
Option "-a" is unknown, try "ip -help".

```

**Comment by EmilioCristalli** [comment]
hi @tameem92, what version of the gem are you using?

I was having the same issue and seems that's been fixed once I upgraded to 2.1.5 (probably fixed in this commit https://github.com/sporkmonger/uuidtools/commit/3cdad0a1a0a9546bcddf7235906d4179c87eb03e)

**Comment by tameem92** [comment]
Hi @EmilioCristalli Thanks for your help. Upgrading to 2.1.5 has resolved my issue as well.
